addons/mail/tests/test_mail_message_translate.py

```diff
@@ -81,11 +81,16 @@ def test_update_message(self):
         self.assertEqual(self.env["mail.message.translation"].search_count([]), 1)
         # The translation records should not be discarded if the body did not change.
         self.make_jsonrpc_request(
-            "/mail/message/update_content", {"message_id": self.message.id, "body": None, "attachment_ids": []}
+            "/mail/message/update_content",
+            {"message_id": self.message.id, "update_data": {"attachment_ids": [], "body": None}},
```

Comment:
Reading this, I don't see what you gain, expect you blur everything using sub-dicts.